packages/core/content-manager/server/src/history/services/history.ts

```diff
@@ -167,7 +167,12 @@ const createHistoryService = ({ strapi }: { strapi: Core.Strapi }) => {
             : { documentId: context.params.documentId, locale: context.params?.locale };
 
         const defaultLocale = await getDefaultLocale();
-        const locale = documentContext.locale || defaultLocale;
+
+        // TODO
```

Comment:
Hey @markkaylor 

I was having trouble with content history during the bulk locale publishing stuff I'm working on. This bit of code would cause an error to be thrown from

https://github.com/strapi/strapi/blob/b9f98c5dd7d398642b83d37a9024084add4c0af3/packages/core/core/src/services/document-service/internationalization.ts#L47

I'm not 100% what the best solution is here. I think the problem is that now publishing can accept multiple locales, history will intercept those calls and try and proceed with the logic below using an array of locales from the params. docService.findOne then throws as this is an invalid param.

What do you think the best idea here is?

Reply:
Maybe we can just check if it's an array and skip it for now 🤷‍♂️ ? Then we can circle back to adjust the logic here to handle all bulk actions. What do you think? cc @remidej

Reply:
I think I agree with you @markkaylor. this is a place where we need to eventually write proper tests, and we should probably be the ones to write it, so I think opting out for now in this branch should be okay